README.md

```diff
@@ -1 +1,17 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello Microverse 
```

Comment:
- [ ] I would suggest using the [README template](https://github.com/microverseinc/readme-template) because it is a requirement for this project.
![image](https://user-images.githubusercontent.com/81522435/143043828-f5e4c8e3-ed86-4176-ac21-330e2788d3d4.png)
You could add your current README information into the [README template](https://github.com/microverseinc/readme-template) and customize it for your project :wink:

Reply:
I have updated the README file based on the template given and removed some part of it which are not that much required for this project.